webapp/src/DataFixtures/DefaultData/LanguageFixture.php

```diff
@@ -0,0 +1,72 @@
+            ['awk',    'awk',        'AWK',         ['awk'],                     false, null,         false,  true,   1,            'awk'],
+            ['bash',   'bash',       'Bash shell',  ['bash'],                    false, 'Main file',  false,  true,   1,            'bash'],
+            ['c',      'c',          'C',           ['c'],                       false, null,         true,   true,   1,            'c'],
+            ['cpp',    'cpp',        'C++',         ['cpp', 'cc', 'cxx', 'c++'], false, null,         true,   true,   1,            'cpp'],
+            ['csharp', 'csharp',     'C#',          ['csharp', 'cs'],            false, null,         false,  true,   1,            'csharp'],
+            ['f95',    'f95',        'Fortran',     ['f95', 'f90'],              false, null,         false,  true,   1,            'f95'],
+            ['hs',     'haskell',    'Haskell',     ['hs', 'lhs'],               false, null,         false,  true,   1,            'hs'],
+            ['java',   'java',       'Java',        ['java'],                    false, 'Main class', true,   true,   1,            'java_javac_detect'],
+            ['js',     'javascript', 'JavaScript',  ['js'],                      false, 'Main file',  false,  true,   1,            'js'],
+            ['lua',    'lua',        'Lua',         ['lua'],                     false, null,         false,  true,   1,            'lua'],
+            ['kt',     'kotlin',     'Kotlin',      ['kt'],                      true,  'Main class', false,  true,   1,            'kt'],
+            ['pas',    'pascal',     'Pascal',      ['pas', 'p'],                false, 'Main file',  false,  true,   1,            'pas'],
+            ['pl',     'pl',         'Perl',        ['pl'],                      false, 'Main file',  false,  true,   1,            'pl'],
+            ['plg',    'prolog',     'Prolog',      ['plg'],                     false, 'Main file',  false,  true,   1,            'plg'],
+            ['py2',    'python2',    'Python 2',    ['py2', 'py'],               false, 'Main file',  false,  true,   1,            'py2'],
+            ['py3',    'python3',    'Python 3',    ['py3'],                     false, 'Main file',  true,   true,   1,            'py3'],
+            ['r',      'r',          'R',           ['R'],                       false, 'Main file',  false,  true,   1,            'r'],
+            ['rb',     'ruby',       'Ruby',        ['rb'],                      false, 'Main file',  false,  true,   1,            'rb'],
+            ['scala',  'scala',      'Scala',       ['scala'],                   false, null,         false,  true,   1,            'scala'],
+            ['sh',     'sh',         'POSIX shell', ['sh'],                      false, 'Main file',  false,  true,   1,            'sh'],
+            ['swift',  'swift',      'Swift',       ['swift'],                   false, 'Main file',  false,  true,   1,            'swift'],
+        ];
```

Comment:
Is this something we should somehow deduplicate with the executable fixture?

Reply:
Any suggestions on how to do this? I could make one `LanguageAndExecutableFixture` but then what to do with `java_javac`, `compare` and `run`?

Reply:
Or a Language fixture that inserts everything needed for that language (Language entity + executable) and a separate Executable fixture that only inserts the compare and run ones?

Reply:
I like it. We will have some duplication in creating the language itself and the createImmutableExecutable method, but that is only minimal.

Reply:
Done, this looks way better.